server/src/main/scala/org/http4s/server/middleware/ChunkAggregator.scala

```diff
@@ -35,7 +35,7 @@ object ChunkAggregator {
       resp: Response[G],
       len: Long): Response[G] =
     resp.transformHeaders { headers =>
-      val hs = headers.flatMap {
+      val hs = Headers(headers.toList.flatMap {
```

Comment:
This seems like one of the stronger arguments for ceasing to be a standard collection: it's not real easy to predict what type a `flatMap` returns in the old implementation.